pints/toy/_logpdfs.py

```diff
@@ -0,0 +1,189 @@
+#
+# Various toy pdfs to use to test samplers
+#
+# This file is part of PINTS.
+#  Copyright (c) 2017, University of Oxford.
+#  For licensing information, see the LICENSE file distributed with the PINTS
+#  software package.
+#
+#
+from __future__ import absolute_import, division
+from __future__ import print_function, unicode_literals
+import numpy as np
+from scipy.stats import multivariate_normal
+
+
+class LogPDF(object):
+    """
+    Class for log pdfs. These are typically
+    toy problems where the target density
+    is known.
+    """
+    def __init__(self, dimension):
+        self._dimension = dimension
+
```

Comment:
If we want to keep this, maybe add `def __call__(self, x)` as `raise NotImplementedError`?